.github/workflows/tests.yml

```diff
@@ -29,7 +29,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [pypy3, 3.6, 3.7, 3.8, 3.9]
+        python-version: [pypy-3.6, 3.6, 3.7, 3.8, 3.9]
```

Comment:
@hukkinj1 should this maybe be pypy-3.7 or 3.8, since 3.6 will be dropped at some point?